service/history/shard/compatibility.go

```diff
@@ -62,7 +61,7 @@ func loadShardInfoCompatibilityCheckWithoutReplication(
 
 		queueCategory, ok := tasks.GetCategoryByID(queueCategoryID)
 		if !ok {
-			panic(fmt.Sprintf("unable to find queue category by queye category ID: %v", queueCategoryID))
+			continue
```

Comment:
this essentially deletes the ack levels in the shard info?

Reply:
yes, load will not propagate the info from ack level to queue state, queue state will also skip storing